openquake/server/tests/views_test.py

```diff
@@ -122,8 +129,16 @@ def setUpClass(cls):
         env['OQ_DISTRIBUTE'] = 'no'
         # let's impersonate the user openquake, the one running the WebUI:
         # we need to set LOGNAME on Linux and USERNAME on Windows
-        env['LOGNAME'] = env['USERNAME'] = 'openquake'
+        env['LOGNAME'] = env['USERNAME'] = username = 'openquake'
```

Comment:
Let's create a Django user called "django-test-user" and not "openquake".

Reply:
Ok

Reply:
Done